torch/distributions/weibull.py

```diff
@@ -0,0 +1,49 @@
+from numbers import Number
+import math
+import torch
+from torch.distributions import constraints
+from torch.distributions.exponential import Exponential
+from torch.distributions.transformed_distribution import TransformedDistribution
+from torch.distributions.transforms import AffineTransform, PowerTransform
+from torch.distributions.utils import _finfo, broadcast_all
```

Comment:
no need for `_finfo`, right?

Reply:
I remember seeing a Uniform based transformed distribution, probably that is why it is there.

Reply:
But that doesn't appear to be the case, so yes, I think it can be removed.

Reply:
Sorry, didn't notice the review earlier.

Reply:
Right, thanks!